### ISSUE TITLE
Problem with screen brightness control

### ISSUE THREAD

**Issue by aksharlabs** [LingmoOS/LingmoOS#6]
Screen brightness can not controll from setting and top pannel
![17093805332462924430127931229667](https://github.com/LingmoOS/LingmoOS/assets/74598801/585d386a-4203-4226-8414-dcc090ac6d58)

**Comment by ganyuanzhen** [comment]
Yes, we'll investigate the compatibility problem with NVIDIA display cards.

**Comment by ganyuanzhen** [comment]
This may not be a issue for the control center.

Free to comment if have any updates.
